pom.xml

```diff
@@ -156,6 +156,10 @@
     <commons.collections.version>3.2.2</commons.collections.version>
     <scala.version>2.12.7</scala.version>
     <scala.binary.version>2.12</scala.binary.version>
+    <scalafmt.version>1.5.1</scalafmt.version>
```

Comment:
If these are all just used in one place, in the plugin config, I'd just inline them below to keep it together. If they get reused in more than one place and have to stay synced, then this makes sense.

Reply:
I moved the scalafmt version inline.  The others I think need to stay as parameters so the shell wrapper can override them, to distinguish between the use cases of build pipeline vs contributor fixing code.